bin/replugged.js

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env node
+/* eslint-disable consistent-return */
```

Comment:
why

Reply:
Lines `34` and `46` give an eslint error because "no returning in arrow function". I'll change it to regular function ig